brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-973 All block Icon - Resolved block icon position issue in editor

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-973

### Screenshots

- Issue - https://d.pr/i/gKP2AT
- Fix - https://d.pr/v/DCqwHt

### Types of changes




- Improvement

### How has this been tested?

- Go to the editor.
- Click on + icon to check Spectra block on Post/Page.
- Inspect any Spectra block SVG and check it is in center or not.

### Checklist:
